snippets/lwc-xml.json

```diff
@@ -0,0 +1,72 @@
+{
+    "Api Version": {
+        "body": [
+            "<apiVersion>$0</apiVersion>"
+        ],
+        "description": "Api Version",
+        "prefix": [
+            "apiVersion"
+        ],
+        "scope": "xml"
+    },
+    "Capability": {
```

Comment:
Add `LWC` before each key `LWC Capability` so user know, that should not use it in apex XML.